pdr_backend/subgraph/prediction.py

```diff
@@ -63,6 +66,7 @@ def mock_prediction(prediction_tuple: tuple) -> Prediction:
         timestamp=timestamp,
         source=source,
         payout=payout,
+        address=address,
```

Comment:
I am no sure about the putting the mock data into the class definition file, but I see the same approach on Idiom-bytes's  PR too. Maybe we can talk about it later

Reply:
I think we should have the contract address somewhere

Reply:
@kdetry 
The mock data implementation was initiated by trent. I'm supporting it for now while we come up with a better structure.

@KatunaNorbert 
1. the id signature comes from the contract/subgraph. this looks correct to me
2. i agree that the contract address should be fetched and stored locally. We'll need to be able to rebuild the ID locally.